StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableView.swift

```diff
@@ -33,19 +33,21 @@ protocol InstitutionTableViewDelegate: AnyObject {
 final class InstitutionTableView: UIView {
 
     private let allowManualEntry: Bool
+    private let institutionSearchDisabled: Bool
     let tableView: UITableView
     private let dataSource: UITableViewDiffableDataSource<Section, FinancialConnectionsInstitution>
-    private lazy var didSelectManualEntry: (() -> Void)? = {
-        return allowManualEntry
-            ? { [weak self] in
-                guard let self = self else { return }
-                self.delegate?.institutionTableView(
-                    self,
-                    didSelectManuallyAddYourAccountWithInstitutions: self.institutions
-                )
-            } : nil
-    }()
     weak var delegate: InstitutionTableViewDelegate?
+    weak var searchBarContainerView: UIView? {
```

Comment:
ignorable: think its worth a comment up here that this will become the section 0's header in the tableview? 🤷‍♂️

Reply:
Added! 

Note that some of the "section 0" checks are just defensive programming in the case that there are ever more sections in the future (there is only one section at the moment)

Your comment sort of alludes to this, but it's quite likely that this code could be restructured because its changed ~3+ times due to evolving features. There's like some "sub-optimality" around how all the pieces interact